### PR TITLE
Add release and version support

### PR DESCRIPTION
**Pull request by DoriftoShoes**

### CODE REVIEW

File: actions/st2_bootstrap_test.meta.yaml [hunk 1]
```diff
@@ -30,7 +30,14 @@
...
       default: "default"
       type: "string"
-    version: 
+    version:
+      type: "string"
+      default: ""
+    version_arg:
+      type: "string"
+      immutable: true
+      default: "{% if version != '' %}--version={{version}}{% endif %}
```
Comment (manasdk) [inline actions/st2_bootstrap_test.meta.yaml]: `"` seems to missing at the end

**Comment by manasdk** [comment]
:+1:

**Comment by Kami** [comment]
:+1:

**Comment by lakshmi-kannan** [comment]
I'll merge this and test it out.

**Pull request merged**